util/container/Dockerfile

```
@@ -0,0 +1,45 @@
+#---------------------
+# Ubuntu 22.04 base
+#---------------------
+FROM ubuntu:22.04
+
+#---------------------
+# Install necessary packages
+#---------------------
+RUN apt-get -y update
+RUN apt-get -y install git
+RUN apt-get -y install autoconf help2man perl python3 make flex bison g++ 
+RUN apt-get -y install libfl2  libfl-dev
+RUN apt-get -y install python3 pip
```

```suggestion
RUN apt-get update && apt-get -y install git autoconf help2man perl python3 python3-pip make flex bison g++ libfl2 libfl-dev
```
Multiple `RUN` commands create multiple file system layers in the container image.
I think in this case it makes sense to not do that and run all of this stuff in one go.
See also here:
https://stackoverflow.com/questions/47079114/should-i-minimize-the-number-of-docker-layers

Also, right now you are installing python3 twice, and i've changed `pip` to `python3-pip` to get rid of a of a warning.

---

Ah I didn't know this. OK! Will add this :D